### PR TITLE
Fixes for recent strings

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

**Comment by Changaco** [comment]
In my opinion none of the proposed changes is necessary, and most of them would actually make things worse.

File: liberapay/constants.py [hunk 1]
```diff
@@ -344,8 +344,8 @@ def __missing__(self, currency):
 )
 
 PRIVACY_FIELDS = OrderedDict([
-    ('hide_giving', (_("Do not publish the amounts of money I send."), False)),
-    ('hide_receiving', (_("Do not publish the amounts of money I receive."), False)),
+    ('hide_giving', (_("Do not publish the amount of money I send."), False)),
```
Comment (jorgesumle) [inline liberapay/constants.py:347]: https://english.stackexchange.com/questions/254339/when-to-use-amount-vs-amounts

**Comment by jorgesumle** [comment]
Using dots instead of commas in some places may be better, I would also write that way. But I am not sure that using commas there is wrong. I wouldn't change anything unless I'm sure it's wrong as it is.

File: liberapay/constants.py (hunk 1 above)
Comment (comradekingu) [reply liberapay/constants.py:347]: Yes. This is also wrong in "Source code of the recurrent donations platform Liberapay"

File: templates/macros/your-tip.html [hunk 2]
```diff
@@ -236,17 +236,17 @@ <h5 class="list-group-item-heading">{{ _("Manual renewal") }}</h5>
     % endif
     % if patron_visibilities == 1
     <p class="text-info">{{ glyphicon('info-sign') }} {{ _(
-        "{username} has chosen not to see who their patrons are, so your donation will be secret.",
```
Comment (comradekingu) [inline templates/macros/your-tip.html:239]: This is a known singular, where "their" would denote plural.

File: templates/macros/your-tip.html [hunk 3]
```diff
@@ -236,17 +236,17 @@ <h5 class="list-group-item-heading">{{ _("Manual renewal") }}</h5>
...
+        "{username} has chosen not to see own patrons, so your donation will be secret.",
         username=tippee_name
     ) }}</p>
     % elif patron_visibilities == 2
     <p class="text-info">{{ glyphicon('info-sign') }} {{ _(
-        "This donation won't be secret, you will appear in {username}'s private list of patrons.",
```
Comment (comradekingu) [inline templates/macros/your-tip.html:244]: Sentence splicing.

File: templates/macros/your-tip.html [hunk 4]
```diff
@@ -300,7 +300,7 @@ <h5 class="list-group-item-heading">{{ _("Public donation") }}</h5>
     </ul>
     % else
     <p class="text-muted">{{ glyphicon('info-sign') }} {{ _(
-        "{username} hasn't yet specified whether they want to see who their patrons are, "
```
Comment (comradekingu) [inline templates/macros/your-tip.html:303]: Singular known they/their is a poor choice of words.

File: www/%username/giving/index.html.spt [hunk 5]
```diff
@@ -227,8 +227,8 @@ next_payday = compute_next_payday_date()
 % elif show_incomplete_profile_warning
     <div class="text-info">
         {{ glyphicon('info-sign') }} {{ _(
-            "Since you've chosen to make a public donation, we recommend that you "
```
Comment (comradekingu) [inline www/%username/giving/index.html.spt:230]: Run-On sentence

File: www/%username/giving/pay/stripe/%payin_id.spt [hunk 6]
```diff
@@ -311,7 +311,7 @@ title = _("Funding your donations")
         ) }}</p>
         % if not route.has_been_charged_successfully()
         <p class="alert alert-warning">{{ _(
-            "Your bank can reject this payment. We recommend sending a copy of "
```
Comment (comradekingu) [inline www/%username/giving/pay/stripe/%payin_id.spt:314]: "We" is invoked without merit. It adds complexity.

File: www/%username/patrons/index.spt [hunk 7]
```diff
@@ -76,10 +76,10 @@ subhead = _("Patrons")
 % if patron_visibilities > 1
 <h3>{{ _("Data export") }}</h3>
 <p><a href="{{ participant.path('patrons/export.csv?scope=active') }}">{{ fontawesome('download') }} {{
-    _("Download the list of currently active patrons")
```
Comment (comradekingu) [inline www/%username/patrons/index.spt:79]: "currently" is redundant. Am I missing something here?

File: www/%username/patrons/index.spt [hunk 8]
```diff
@@ -76,10 +76,10 @@ subhead = _("Patrons")
...
+    _("Download the list of active patrons")
 }}</a></p>
 <p><a href="{{ participant.path('patrons/export.csv?scope=all') }}">{{ fontawesome('download') }} {{
-    _("Download the record of all patrons in the last ten years")
```
Comment (comradekingu) [inline www/%username/patrons/index.spt:82]: "In" is superfluous. It also makes it more of a garden-path sentence.

File: www/%username/patrons/index.spt [hunk 9]
```diff
@@ -46,27 +46,27 @@ subhead = _("Patrons")
...
-            "then {link_start}click here to disable non-anonymous donations{link_end}.",
+            "then {link_start}click here to turn off non-anonymous donations{link_end}.",
             link_start='<button class="link" name="see_patrons" value="no">'|safe,
             link_end='</button>'|safe
         ) }}</p>
     % else
         <p class="text-muted">{{ _(
             "You've chosen not to see who your patrons are. If you change your mind, "
-            "then {link_start}click here to enable non-anonymous donations{link_end}.",
+            "then {link_start}click here allow non-anonymous donations{link_end}.",
```
Comment (comradekingu) [inline www/%username/patrons/index.spt:69]:
```suggestion
            "then {link_start}click here to allow non-anonymous donations{link_end}.",
```

File: www/%username/patrons/index.spt [hunk 10]
```diff
@@ -46,27 +46,27 @@ subhead = _("Patrons")
...
             "You have opted-in to see who your patrons are. If you change your mind, "
-            "then {link_start}click here to disable non-anonymous donations{link_end}.",
+            "then {link_start}click here to turn off non-anonymous donations{link_end}.",
             link_start='<button class="link" name="see_patrons" value="no">'|safe,
             link_end='</button>'|safe
         ) }}</p>
     % else
         <p class="text-muted">{{ _(
             "You've chosen not to see who your patrons are. If you change your mind, "
-            "then {link_start}click here to enable non-anonymous donations{link_end}.",
```
Comment (comradekingu) [inline www/%username/patrons/index.spt:69]: It isn't clear what "enable" would do here.

File: www/%username/patrons/index.spt [hunk 11]
```diff
@@ -46,27 +46,27 @@ subhead = _("Patrons")
     <input type="hidden" name="csrf_token" value="{{ csrf_token }}" />
     % if patron_visibilities == 0
         <p class="text-info">{{ glyphicon('info-sign') }} {{ _(
-            "Liberapay now supports non-anonymous donations, do you want to know "
+            "Liberapay now supports non-anonymous donations. Do you want to know "
             "who your patrons are?"
         ) }}</p>
         <p class="buttons">
             <button class="btn btn-primary" name="see_patrons" value="yes">{{ _(
-                "Enable non-anonymous donations"
```
Comment (comradekingu) [inline www/%username/patrons/index.spt:54]: Presumably this can't be disabled and on, so "enable" adds complexity.

**Comment by comradekingu** [comment]
@Changaco @jorgesumle 
I added some rationale for the rest of the strings.
Most are repeat errors, so it is mostly explained once for each type.

File: templates/macros/your-tip.html (hunk 2 above)
Comment (jorgesumle) [reply templates/macros/your-tip.html:239]: Liberapay uses [singular they](https://en.wikipedia.org/wiki/Singular_they).

Comment (comradekingu) [reply templates/macros/your-tip.html:239]: Hence why I made this PR to fix it.

**Comment by jorgesumle** [comment]
@comradekingu, thanks for the explanation. I think that the only error is the comma splice (after reading [Comma Splice Examples and How to Fix Them](https://examples.yourdictionary.com/comma-splice-examples.html), I think that we need to fix that). The other corrections you proposed are mainly because you prefer simpler and more formal English, but I think that our more formal and verbose style is fine.

**Comment by comradekingu** [comment]
@jorgesumle I proposed them because they are better, in ways I explained.
Simpler is always better when it means doing away with needless complexion.
I don't prefer less formal English. The examples of errors are not more formal, and some are unacceptable in any setting.

**Comment by comradekingu** [comment]
@Changaco It is just a matter of avoiding issues. As an example "As Martin was a member of the board, they decided against the decision to allow the deconstruction." would be incomprehensible without interpreting as the language dictates.
Even if it didn't actually create these kinds of errors, conditions where a common pronoun is used for both sexes are exactly the same as everywhere else in Norway. There is no better control group or example in the world.

It is like changing the master branch naming to "main", only now the main lingua franca gets a lot harder to learn for people.

**Pull request closed without merging**